packages/transformers/html/src/dependencies.js

```diff
@@ -122,19 +122,26 @@ export default function collectDependencies(
     seen.add(node);
 
     if (tag === 'meta') {
-      if (
-        !Object.keys(attrs).some(attr => {
-          let values = META[attr];
-          return (
-            values &&
-            values.includes(attrs[attr]) &&
-            attrs.content !== '' &&
-            !(attrs.name === 'msapplication-config' && attrs.content === 'none')
-          );
-        })
-      ) {
-        return node;
+      const isMetaDependency = Object.keys(attrs).some(attr => {
+        let values = META[attr];
+        return (
+          values &&
+          values.includes(attrs[attr]) &&
+          attrs.content !== '' &&
+          !(attrs.name === 'msapplication-config' && attrs.content === 'none')
+        );
+      });
+      if (isMetaDependency) {
+        const metaAssetUrl = attrs.content;
+        if (metaAssetUrl && !/^(http|https):\/\//.test(metaAssetUrl)) {
```

Comment:
Seems ok but don't we usually let resolvers handle full URLs? If they don't resolve to anything the original specifier gets put back. What was happening here?

Reply:
Just forgot we did that, will remove it